build-site.sh

```diff
@@ -1,23 +1,24 @@
 #!/bin/bash
 
 if [ -z "$JEKYLLSITEBUILD" ]; then
-        export JEKYLLSITEBUILD=latest
+    export JEKYLLSITEBUILD=latest
 fi
 
 if [ -z "$JEKYLL_ENV" ]; then
-	export JEKYLL_ENV=staging
+    export JEKYLL_ENV=staging
 fi
+
 docker run \
-  --cap-drop ALL \
-  --rm \
-  -it \
-  -p 4000:4000 \
-  -e JEKYLL_ACTION \
-  -e JEKYLL_CONFIG \
-  -e JEKYLL_ENV \
-  -v /etc/passwd:/etc/passwd:ro \
-  -v /etc/group:/etc/group:ro \
-  -u "$(id -u)":"$(id -g)" \
-  -v "$(pwd)":/srv/source \
-  linaroits/jekyllsitebuild:"$JEKYLLSITEBUILD" \
-  build-site.sh
+    --cap-drop ALL \
+    --rm \
+    -it \
```

Comment:
```suggestion
    -t \
```
This may be stale: see https://github.com/ArmNNWebsite/website/pull/33. I'm having trouble following where @pcolmer  is modifying these.

Reply:
Yes, the script should be using `-t` now and not `-it`. The `-i` isn't needed for users (since they aren't interacting with the underlying stuff going on) and it causes Bamboo to break when it uses this script to test changes made to the build container.

Reply:
Now that I've got the `-t` vs `-it` bit tested and resolved (I was off for a couple of days), I'm back to going through all of the web site repos **again** to make sure they are all correct and up to date :)